routes.js

```diff
@@ -0,0 +1,57 @@
+const express = require('express');
+
+const router = express.Router();
+const low = require('lowdb');
+
+const db = low('db/db.json', { storage: require('lowdb/lib/storages/file-async') }); // eslint-disable-line
+
+router.get('/movies', (req, res) => {
```

Comment:
Because all of these routes start with `/movies` I would prefer to see you mount the router on `/movies` instead.

So in `app.js` you would have `app.use("/movies", router);` and here you would have `router.get("/", ...`